torch/distributed/elastic/multiprocessing/__init__.py

```diff
@@ -239,14 +240,25 @@ def start_processes(
             error_files[local_rank] = os.devnull
             envs[local_rank]["TORCHELASTIC_ERROR_FILE"] = ""
         else:
-            clogdir = os.path.join(log_dir, str(local_rank))
-            os.mkdir(clogdir)
+            if log_for_fb_tupperware:
+                clogdir = log_dir
+            else:
+                clogdir = os.path.join(log_dir, str(local_rank))
+                os.mkdir(clogdir)
+
+            if log_for_fb_tupperware is not None:
```

Comment:
I wonder if we can make this FB specific using specific implementation using buck build tricks that use internal fb patches?